ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -285,6 +285,15 @@ public LogicalSchema withoutMetaAndKeyFieldsInValue() {
         alias);
   }
 
+  public LogicalSchema withoutMetaAndKeyFields() {
```

Comment:
We shouldn't be doing this.... I've a fix for the bug here: https://github.com/confluentinc/ksql/pull/3043

Reply:
Ditto